### PR TITLE
rename packages to targets

### PR DESCRIPTION
**Pull request by evgeni**
we're not always working with packages in obal, and in future we might
also work on other things with the same framework

### CODE REVIEW

File: obal/__init__.py [hunk 1]
```diff
@@ -214,41 +222,41 @@ def ansible_config_path(cls):
         return os.environ.get('OBAL_ANSIBLE_CFG', os.path.join(cls.data_path(), 'ansible.cfg'))
 
 
-def find_playbooks(playbooks_path):
+def find_playbooks(application_config):
```
Comment (ekohl) [inline obal/__init__.py]: Should this become a method on the `ApplicationConfig`?

File: tests/test_playbooks.py [hunk 2]
```diff
@@ -14,14 +14,14 @@ def playbook_id(fixture_value):
     return fixture_value.name
 
 
-@pytest.fixture(params=obal.find_playbooks(obal.ApplicationConfig.playbooks_path()), ids=playbook_id)
+@pytest.fixture(params=obal.find_playbooks(obal.ApplicationConfig, ids=playbook_id))
```
Comment (ekohl) [inline tests/test_playbooks.py]:
```suggestion
@pytest.fixture(params=obal.find_playbooks(obal.ApplicationConfig), ids=playbook_id)
```

File: obal/__init__.py (hunk 1 above)
Comment (evgeni) [reply obal/__init__.py]: like `ac.find_playbooks`? maybe.

Comment (ekohl) [reply obal/__init__.py]: Yes. Previously it was a very clean standalone test but now it's coupled to app config anyway. I also think that finding playbooks is also part of the application config.

**Pull request merged**